### PR TITLE
js-dom

### PR DESCRIPTION
**Pull request by Yasya23**
### js-dom

[Demo](https://yasya23.github.io/js-dom/) |
[Code base](https://github.com/Yasya23/js-dom)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Yasya23** [comment]
It seems to me that I checked everything according to the instructions. Please rewiew.

File: submissions/Yasya23/js-dom/script.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+];
+
+document.querySelector("main").innerHTML = `
+        <div class="img">
+          <div class="img__column">
+            <img
+              src="${images[0].src}" alt="${images[0].alt}"
+            />
+            <img
+              src="${images[10].src}" alt="${images[10].alt}"
```
Comment (madmaxWMFU) [inline submissions/Yasya23/js-dom/script.js:148]: Create render function.
Add initial value to first render to list.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Nice work! See some comments below.

File: submissions/Yasya23/js-dom/script.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
+"use strict";
+
+const images = [
+  {
+    id: "cat",
```
[the remark this answers is not in the document]
Comment (Yasya23) [reply submissions/Yasya23/js-dom/script.js]: Thanks for checking! Fixed it.

File: submissions/Yasya23/js-dom/script.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+const popupIcon = document.querySelector("#popup");
+popupIcon.addEventListener("click", showPopup);
+
+const cat = document.querySelector("#cat");
+const dog = document.querySelector("#dog");
+const horse = document.querySelector("#horses");
+
+cat.addEventListener("click", changeImagesByClick);
+dog.addEventListener("click", changeImagesByClick);
+horse.addEventListener("click", changeImagesByClick);
```
[the remark this answers is not in the document]
Comment (Yasya23) [reply submissions/Yasya23/js-dom/script.js]: Fixed it.

File: submissions/Yasya23/js-dom/script.js (hunk 1 above)
Comment (Yasya23) [reply submissions/Yasya23/js-dom/script.js]: I'm not sure I did it right.

**Comment by Yasya23** [comment]
Thanks for the help!

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Pull request merged**